tests/pr_test.go

```diff
@@ -134,3 +136,50 @@ func TestRunBasicExampleWithFlavor(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunStandardSolution(t *testing.T) {
+	t.Parallel()
+
+	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  standardSolutionTerraformDir,
+		Region:        "us-south",
+		Prefix:        "postgres-st-da",
+		ResourceGroup: resourceGroup,
+	})
+
+	options.TerraformVars = map[string]interface{}{
+		"pg_version":                "16", // Always lock this test into the latest supported PostgreSQL version
+		"existing_kms_instance_crn": permanentResources["hpcs_south_crn"],
+		"kms_endpoint_type":         "public",
+		"resource_group_name":       options.Prefix,
+	}
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunStandardUpgradeSolution(t *testing.T) {
```

Comment:
Hi @ocofaigh - I think we require the `TestRunUpgradeCompleteExample` because this covers upgrade tests around creation of CBR, VPC, Security groups, VPE

Reply:
We dont need to upgrade test VPC, Security groups or VPE - the module doesn't create those resources (they are only created in the example). As for CBR, I don't think its enough to have a 2nd upgrade test IMHO. I think the risk is low anyway, and probably some day we will add CBR support to the DA anyway

Reply:
Sure Conall, I have removed the upgrade test.